applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -99,14 +99,270 @@ namespace Kratos
 	void CrBeamElement3D2N::Initialize() {
 
 		KRATOS_TRY;
-		if (this->mIterationCount == 0)
+		KRATOS_CATCH("")
+	}
+
+	void CrBeamElement3D2N::GetSecondDerivativesVector(Vector& rValues, int Step)
+	{
+
+		KRATOS_TRY
+		if (rValues.size() != msElementSize) rValues.resize(msElementSize, false);
+
+		for (int i = 0; i < msNumberOfNodes; ++i)
+		{
+			int index = i * msDimension * 2;
+
+			rValues[index] = this->GetGeometry()[i]

```

Comment:
...here you would have best done

      const auto& acc = his->GetGeometry()[i].FastGetSolutionStepValue(ACCELERATION,step)
      rvalues[index] = acc[0];
      rvalues[index+1] = acc[1];
      rvalues[index+2] = acc[2];

same for angular acceleration